zerver/lib/email_validation.py

```diff
@@ -160,20 +163,23 @@ def process_email(email: str) -> None:
         deactivated = not existing_user_profile.is_active
 
         if existing_user_profile.is_active:
```

Comment:
This line would probably be better as `if not deactivated` (or better, swap the if statement and do `if deactivated`.

Reply:
Can we add this is as a cleanup commit?  I think there's some readability in the handling the common case first, and some sort of double-negative is kinda hard to avoid here, so I'm a little wary to tweak this too much from the original code.

Reply:
Yeah definitely.